src/nearcache/RepairingHandler.ts

```diff
@@ -26,7 +26,6 @@ import {NearCache} from './NearCache';
 export class RepairingHandler {
 
     private readonly nearCache: NearCache;
-    private readonly partitionCount: number;
     private readonly partitionService: PartitionServiceImpl;
```

Comment:
Why we use PartitionServiceImpl instead of PartitionService here? Just asking.

Reply:
It could be an artifact from older versions. `PartitionService` interface should be enough here.